### PR TITLE
Fix bug with loading molecule statmech during Rate jobs

### PR DESCRIPTION
**Pull request by mjohnson541**
This causes ARC to still load statmech for species it isn't calculating thermo for.

### CODE REVIEW

File: arc/processor.py [hunk 1]
```diff
@@ -198,10 +199,10 @@ def process(self):
                     species.rmg_thermo = self.rmgdb.thermo.getThermoData(species.rmg_species)
                 except ValueError:
                     logging.info('Could not retrieve RMG thermo for species {0}, possibly due to missing 2D structure '
-                                 '(bond orders). Not including this species in the parity plots.'.format(species.label))
+                                     '(bond orders). Not including this species in the parity plots.'.format(species.label))
```
Comment (alongd) [inline arc/processor.py]: Can you remove this line from the commit?

Comment (mjohnson541) [reply arc/processor.py]: Fixed

**Comment by mjohnson541** [comment]
Rebased

**Comment by alongd** [comment]
Thanks!

**Pull request merged**